runtime/oti/UnsafeAPI.hpp

```diff
@@ -415,7 +415,7 @@ class VM_UnsafeAPI
 	static VMINLINE U_8
 	getBoolean(J9VMThread *currentThread, MM_ObjectAccessBarrierAPI *objectAccessBarrier, j9object_t object, UDATA offset, bool isVolatile)
 	{
-		return (U_8)get32(currentThread, objectAccessBarrier, object, offset, isVolatile, 0, false);
+		return (U_8)(get32(currentThread, objectAccessBarrier, object, offset, isVolatile, 0, false) != 0);
```

Comment:
There's no reason to do this in one line.  More readable (and debuggable!) if done with each line doing one thing.

```c
I_32 value = get32(currentThread, objectAccessBarrier, object, offset, isVolatile, 0, false);
return (U_8)(value != 0);
```